Sources/Sentry/Public/SentryAttachment.h

```diff
@@ -3,6 +3,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Attachment Type
+ */
+typedef NS_ENUM(NSInteger, SentryAttachmentType) {
```

Comment:
I might be missing something, but what is the use of this attachment type? I see a bunch of initializers that set the `attachmentType` property, but it's never actually read anywhere? It seems to be unused?

Reply:
AttachmentType Is a information for the server

Reply:
Good point. I need to use this in the envelope